ocs_ci/ocs/ui/base_ui.py

```diff
@@ -840,6 +849,9 @@ def navigate_object_bucket_claims_page(self):
         self.do_click(
             locator=self.page_nav["object_bucket_claims_page"], enable_screenshot=False
         )
+        from ocs_ci.ocs.ui.mcg_ui import ObcUI
+
+        return ObcUI()
```

Comment:
Why do you return `obcUI` instance on `navigate_object_bucket_claims_page` method?